components/Filter/Index.js

```diff
@@ -0,0 +1,145 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { theme } from "@/styles";
+
+export default function Filter({ onSubmit, showFilterWindow }) {
+  const [author, setAuthor] = useState("");
+  const [category, setCategory] = useState("");
+
+  const handleAuthorChange = (event) => {
+    const value = event.target.value;
+    setAuthor(value);
+  };
+
+  const handleCategoryChange = (event) => {
+    const value = event.target.value;
+    setCategory(value);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    onSubmit({ author, category });
+  };
+
+  return (
+    <StyledFilterForm show={showFilterWindow} onSubmit={handleSubmit}>
+      <StyledFilterBox>
+        <h2>Filter</h2>
+        <StyledCloseButton onClick={() => (window.location.href = "/")}>
```

Comment:
Two things:

1. Is this the intended behaviour ? To always redirect the user to the homepage (where the filter already is anyways) and thus leading to a reload of the page? Would it not make more sense to set the boolean for showing the filter to `false`?
2. working with the window API within a Next Application would not be my first choice, since there is the `useRouter` hook specially designed for Next Apps

Reply:
It wasn't meant to be there at all, I just forgot to delete it ...